cellarium/cas/visualization/__init__.py

```diff
@@ -0,0 +1,10 @@
+import logging
+
+try:
+    from .circular_tree_plot_umap_dash_app.app import CASCircularTreePlotUMAPDashApp  # noqa
+    from .ui_utils import find_and_kill_process  # noqa
+
+    __all__ = ["CASCircularTreePlotUMAPDashApp", "find_and_kill_process"]
+except ImportError:
+    logging.debug("Visualization dependencies not installed")
```

Comment:
Maybe also say:
"To install Cellarium CAS Client with visualation dependencies from PiPy, please try `pip install --force-reinstall cellarium-cas[vis]`".